### ISSUE TITLE
Selecting Multiple Schedules in Schedule Card Breaks Schedule

### ISSUE THREAD

**Issue by grangep** [asantaga/wiserHomeAssistantPlatform#432]
This is reproducible on my system with Wiser Integration 3.4.2
Go to the Wiser Schedule Card.
Select any two Schedules.
Deselect them without attempting to edit or change them.
Hit the Back button.
Go to the Wiser App. Here it can be seen that the two schedules are broken in some way. Usually both schedules are not connected to any rooms, but occasionally two rooms are connected to the same schedule.

**Comment by msp1974** [comment]
Can you explain more, maybe a video.  I dont understand how you are selecting 2 schedules.  If you are on the card with the schedule at the top and you are 'selecting' schedules, this is assigning and deassigning rooms to the schedule.

Therefore if you are deselecting amd leave no assigned room, then you will see what you describe.  This is expected functionality.

It should be noted that the name of your schedules maybe the same as your room names which can be confusing.  Schedule names can be changed to eliviate this.

**Comment by grangep** [comment]
Ah ok I think I get it now. My room names and schedule names are indeed the same, which was helping to confuse me. I'll change them. Plus my explanation was naff, let me try again. Originally this was a mistake on my part. Starting on the schedule card I selected a schedule. This brings up the schedule assignment screen, with the room assigned to that schedule highlighted. At this point I realised I'd selected the wrong schedule, so I mistakenly clicked on another room. Seeing that the two were now selected I thought "oops, back out & start again" so hit the back button. This left one room disconnected from a schedule.
Would it be possible to have some sort of confirmation, like a save button, so that backing out doesn't leave the room disconnected please? I guess that makes this an enhancement request.

**Comment by andyblac** [comment]
@grangep please try the new schedule card / panel in RC1, let me know if any issues...

<img width="1472" height="1110" alt="Image" src="https://github.com/user-attachments/assets/f5d86986-7103-44a7-90f9-45c943559f74" />
<img width="1472" height="1110" alt="Image" src="https://github.com/user-attachments/assets/e86d7ece-a1a9-45fe-bef9-b4ccebe09d69" />

<img width="1472" height="1110" alt="Image" src="https://github.com/user-attachments/assets/67710b1c-8740-464c-b10c-c192d8588834" />

<img width="1472" height="1110" alt="Image" src="https://github.com/user-attachments/assets/877f37f1-23c4-4fb2-acf2-0651117a2e68" />

**Comment by grangep** [comment]
Hi, I'm afraid I'm not using Home Assistant to control Wiser just now.
